roles.yml

```diff
@@ -0,0 +1,31 @@
+---
+_meta:
+  type: "roles"
+  config_version: 2
```

Comment:
Does 2 mean the version of elasticsearch in this context or something else?

Reply:
It's for the OpenDistro For ElasticSearch plugin.

Reply:
Got it - do you think it's worth adding a comment or will be clear to a user from context?

Reply:
See: https://opendistro.github.io/for-elasticsearch-docs/docs/security/configuration/yaml/

They do not provide a detailed explanation for "config_version", so my guess is that it tells the plugins what version of the configuration it is using.

Reply:
Okay, I just meant something simple like:
```
# OpenDistro Config Version For ElasticSearch Plugin
config_version: 2 
```
Unless that's already known to a user of this code.  I'm not sure if it is or isn't.

Reply:
It's in all of their examples with zero explanation, so I don't want to _add_ my own explanation, in case I don't quite get it right.

Reply:
Fair enough!